### ISSUE TITLE
Some axe results are not properly listed in `run`

### ISSUE THREAD

**Issue by ttsukagoshi** [ttsukagoshi/axe-scan#47]
In some web pages, `axe-scan summary` yields summary results where some WCAG SC are regarded as VIOLATION even though no alert appears in `axe-scan run`.
